app-testing/src/pages/device_landing.py

```diff
@@ -45,6 +47,105 @@ def get_lights_toggle(self) -> Optional[WebElement]:
         )
         return self.base.clickable_wrapper(lights, 5)
 
+    def get_robot_image(self) -> Optional[WebElement]:
+        """Get the robot_image."""
+        lights: Element = Element(
+            (By.ID, "RobotCard_opentrons-dev_robotImage"), f"Robot image."
```

Comment:
`make flake8` or `make lint` would catch all the instances where there are `f` strings that are unneeded.  `f` strings are only for when you need the string evaluated at runtime like when you want to print a variable.

- Please do run `make format` so that black runs.
- `make lint` has a lot of errors but there is no reason to resolve until we get through removing old tests that no longer work.
- Please change all the occurrences of  unneeded f strings in files touched by this PR.

[`f` strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) 

```suggestion
            (By.ID, "RobotCard_opentrons-dev_robotImage"), "Robot image."
```

Reply:
Done